draft-ietf-tls-subcerts.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: Delegated Credentials for (D)TLS
 abbrev:
-docname: draft-ietf-tls-subcerts-latest
+docname: draft-ietf-tls-subcerts-15
+submissionType: IETF
+date: January 2023
 category: std
 
 ipr: trust200902
-area: Security
-workgroup:
-keyword: Internet-Draft
+# area: Security
+# workgroup: tls
```

Comment:
```suggestion
```

If these lines are no longer required, just delete them instead of commenting them out.

Reply:
```suggestion
area: Security
workgroup: tls
```

Reply:
We recommend leaving it in because we want this info in the XML file.  We have removed the hash tags so they are longer commented out.